### PR TITLE
more informative DateTimeEnumerator error + test refactoring

### PR DESCRIPTION
**Pull request by inponomarev**
1. More informative error message for the case when DateTimeEnumerator receives out-of-range value.
2. `assertThrows` instead of old dirty try-catch-expect code in all the unit tests.
3. Changed tabs to spaces in some files -- use 'ignore whitespace' option when seeing the diff.

### CODE REVIEW

**Comment by inponomarev** [comment]
using bash / sed scripts
* ensured LF at the end of every .java file,
* removed trailing whitespace, 
* changed tabs for spaces
this should decrease the number of Checkstyle warnings significantly

File: celesta-core/src/test/java/ru/curs/lyra/grid/DateFieldNumeratorTest.java [hunk 1]
```diff
@@ -1,58 +1,88 @@
...
+                        DateFieldEnumerator.MIN.subtract(BigInteger.TEN).longValue()))
+                ,
+                Date.from(
+                        Instant.ofEpochMilli(
+                                DateFieldEnumerator.MAX.add(BigInteger.TEN).longValue()))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("wrongDates")
```
Comment (ioanngolovko) [inline celesta-core/src/test/java/ru/curs/lyra/grid/DateFieldNumeratorTest.java]: Why? It's easier to write `@MethodSource` and name static method as `outOfRangeVaules` for example.

File: celesta-sql/src/test/java/ru/curs/celesta/score/VersionStringTest.java [hunk 2]
```diff
@@ -1,86 +1,75 @@
...
+package ru.curs.celesta.score;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class VersionStringTest {
+
+    @Test
+    public void test1() {
```
Comment (ioanngolovko) [inline celesta-sql/src/test/java/ru/curs/celesta/score/VersionStringTest.java]: It's a bad name for the test case. Provide correct naming and `@DisplayName` if it helps.

File: celesta-core/src/main/java/ru/curs/lyra/grid/DateFieldEnumerator.java [hunk 3]
```diff
@@ -4,14 +4,15 @@
 import java.util.Date;
 
 public class DateFieldEnumerator extends KeyEnumerator {
+    public static final long MIN_TIMESTAMP = -2208988800000L;
```
Comment (ioanngolovko) [inline celesta-core/src/main/java/ru/curs/lyra/grid/DateFieldEnumerator.java]: Maybe it's better to calculate through Instant?

**Review by ioanngolovko — APPROVED** [review]

**Pull request merged**